config/locales/en.yml

```
@@ -61,6 +61,48 @@ en:
     label:
       user:
         email: Email address
+  pages:
+    home:
+      hero: The POD Aggregator is an open source system to aggregate MARC bibliographic and holdings data for institutions that participate in the Platform for Open Data (POD) initiative.
+      providers:
+        heading: POD Providers
+        body_html:
+          zero: 'There are currently no data providers.'
+          one: 'There is currently <b>1</b> data provider. The most recent uploads were made by <b>%{uploaders}</b> on %{upload_date}.'
+          other: 'There are currently <b>%{count}</b> data providers. The most recent uploads were made by <b>%{uploaders}</b> on %{upload_date}.'
+      data:
+        heading: POD Data
+        body_html:
+          zero: 'POD Aggregator currently holds no records.'
+          one: 'POD Aggregator currently holds <b>1 record</b>.'
+          other: 'POD Aggregator currently holds a total of <b>%{total_records} records</b>, of which <b>%{unique_records} are unique</b>.'
+      login:
+        heading: Already a POD user?
+        button: Login
+        contact_heading: Interested in using POD data?
+        contact_html: Check out the <a href='%{wiki_url}'>POD wiki</a>.
```

```suggestion
        contact_html: Check out the <a href='%{wiki_url}'>POD wiki</a> or contact <a href='mailto:pod-support@lists.stanford.edu'>pod-support@lists.stanford.edu</a>.
```

It's fine by me if we want to move that email into an i18n key or some other config option.